packages/ui-library/src/theme/typography.ts

```diff
@@ -4,156 +4,54 @@ import { colors } from './colors';
 
 export const buildTypographyVariant = (mode: PaletteMode): TypographyOptions => ({
   fontFamily: "'Inter', sans-serif",
-  displayXl: {
-    fontFamily: 'Space Grotesk',
-    fontSize: '12.5rem', // 200/16
-    fontStyle: 'normal',
-    fontWeight: 700,
-    lineHeight: 0.64, // 128/200
-    letterSpacing: '-0.005em', // -1/200
-  },
-  displayLg: {
-    fontFamily: 'Space Grotesk',
-    fontSize: '8rem', // 128/16
-    fontStyle: 'normal',
-    fontWeight: 700,
-    lineHeight: 0.5, // 64/128
-    letterSpacing: '-0.0078125em', // -1/128
-  },
-  displayMd: {
-    fontFamily: 'Space Grotesk',
-    fontSize: '6rem', // 96/16
-    fontStyle: 'normal',
-    fontWeight: 700,
-    lineHeight: 1, // 96/96
-  },
-  displayXs: {
-    fontFamily: 'Space Grotesk',
-    fontSize: '5rem', // 80/16
-    fontStyle: 'normal',
-    fontWeight: 700,
-    lineHeight: 'normal',
-    letterSpacing: '-0.0125em', // -1/80
-  },
-  h1: {
-    fontFamily: 'Inter',
-    fontSize: '3.3125rem', // 53/16
-    fontStyle: 'normal',
-    fontWeight: 400,
-    lineHeight: 1.20755, // 64/53
-    letterSpacing: '-0.0188679em', // -1/53
-  },
   h1Bold: {
     fontFamily: 'Inter',
-    fontSize: '3.3125rem', // 53/16
+    fontSize: '2.5rem', // 40/16
     fontStyle: 'normal',
-    fontWeight: 700,
-    lineHeight: 1.20755, // 64/53
-    letterSpacing: '-0.0188679em', // -1/53
-  },
-  h2: {
-    fontFamily: 'Inter',
-    fontSize: '2.6875rem', // 43/16
-    fontStyle: 'normal',
-    fontWeight: 400,
-    lineHeight: 1.2093, // 52/43
-    letterSpacing: '-0.0232558em', // -1/43
+    fontWeight: 800,
+    lineHeight: 1.2, // 48/40
+    letterSpacing: '-0.025em', // -1/40
+    color: colors[mode].typography.typo1,
   },
   h2Bold: {
     fontFamily: 'Inter',
-    fontSize: '2.6875rem', // 43/16
+    fontSize: '2rem', // 32/16
     fontStyle: 'normal',
     fontWeight: 700,
-    lineHeight: 1.2093, // 52/43
-    letterSpacing: '-0.0232558em', // -1/43
-  },
-  h3: {
-    fontFamily: 'Inter',
-    fontSize: '2.125rem', // 34/16
-    fontStyle: 'normal',
-    fontWeight: 400,
-    lineHeight: 1.17647, // 40/34
-    letterSpacing: '-0.0294118em', // -1/34
+    lineHeight: 1.25, // 40/32
+    letterSpacing: '-0.03125em', // -1/32
   },
   h3Bold: {
     fontFamily: 'Inter',
-    fontSize: '2.125rem', // 34/16
+    fontSize: '1.5rem', // 24/16
     fontStyle: 'normal',
     fontWeight: 700,
-    lineHeight: 1.17647, // 40/34
-    letterSpacing: '-0.0294118em', // -1/34
-  },
-  h4: {
-    fontFamily: 'Inter',
-    fontSize: '1.6875rem', // 27/16
-    fontStyle: 'normal',
-    fontWeight: 400,
-    lineHeight: 1.18519, // 32/27
+    lineHeight: 1.167, // 28/24
+    letterSpacing: '-0.0417em', // -1/24
+    color: colors[mode].typography.typo1,
   },
   h4Bold: {
```

Comment:
can we create variants with mui theme for mobile? I think those bolds are bolder on mobile for the h* components